mobile/mobile.go

```diff
@@ -541,8 +541,8 @@ func (m *Mobile) GetPhotoMetadata(id string) (string, error) {
 	}
 	meta, err := thrd.GetPhotoMetaData(id, block)
 	if err != nil {
-		log.Errorf("get photo meta data failed %s: %s", id, err)
-		return "", err
+		log.Warningf("get photo meta data failed %s: %s", id, err)
+		meta = &model.PhotoMetadata{}
```

Comment:
here's where we just give back empty meta if it hasn't been found instead of an error - allowing the rest of a thread to be loaded